src/modm/math/geometry/angle_int.hpp

```diff
@@ -0,0 +1,67 @@
+// coding: utf-8
+// ----------------------------------------------------------------------------
+/*
+ * Copyright (c) 2024, Thomas Sommer
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+#include <cmath>
+#include <modm/math/utils/integer_traits.hpp>
+#include <numbers>
+
+namespace modm
+{
+
+/// @brief Represents an absolute angle in a full circle
+template<int Bits>
+struct modm_packed IntegerAngle
+{
+	using T = modm::least_uint<Bits>;
+	// @todo need something like modm::least_int<Bits + 1>;
+	using DeltaType = int16_t;
```

Comment:
Something like this?
```suggestion
	using DeltaType = std::make_signed_t<modm::least_uint<Bits + 1>>;
```

Reply:
Thank you! This should do it.